app/__init__.py

```diff
@@ -1,7 +1,9 @@
 from flask import Flask
 
 
-def create_app(test_config=None):
+def create_app():
     app = Flask(__name__)
+    from .routes import planets_bp
+    app.register_blueprint(planets_bp)
```

Comment:
Consider adding a line space to enhance readability

```suggestion
    app = Flask(__name__)
    
    from .routes import planets_bp
    app.register_blueprint(planets_bp)
```